go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -564,6 +564,9 @@ var AlterVReplicationTable = []string{
 	"ALTER TABLE _vt.vreplication ADD KEY workflow_idx (workflow(64))",
 	"ALTER TABLE _vt.vreplication ADD COLUMN rows_copied BIGINT(20) NOT NULL DEFAULT 0",
 	"ALTER TABLE _vt.vreplication ADD COLUMN tags VARBINARY(1024) NOT NULL DEFAULT ''",
+
+	// records the time of the last heartbeat. Heartbeats are only received if the source has no recent events
+	"ALTER TABLE _vt.vreplication ADD COLUMN time_heartbeat BIGINT(20) NOT NULL DEFAULT 0",
```

Comment:
We should remove the default display widths here at some point (could come later), that will cause a warning in 8.0 and fail later on:
```
"ALTER TABLE _vt.vreplication ADD COLUMN time_heartbeat BIGINT NOT NULL DEFAULT 0",
```

```
mysql> select @@version;
+-----------+
| @@version |
+-----------+
| 8.0.27    |
+-----------+
1 row in set (0.00 sec)

mysql> create table uttest (time_heartbeat BIGINT(20) NOT NULL DEFAULT 0);
Query OK, 0 rows affected, 1 warning (0.01 sec)

mysql> show warnings;
+---------+------+------------------------------------------------------------------------------+
| Level   | Code | Message                                                                      |
+---------+------+------------------------------------------------------------------------------+
| Warning | 1681 | Integer display width is deprecated and will be removed in a future release. |
+---------+------+------------------------------------------------------------------------------+
1 row in set (0.00 sec)

mysql> show create table uttest\G
*************************** 1. row ***************************
       Table: uttest
Create Table: CREATE TABLE `uttest` (
  `time_heartbeat` bigint NOT NULL DEFAULT '0'
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
1 row in set (0.00 sec)
```

Meanwhile, 5.7 will add the default display width (20 for this type) automatically.